src/main/java/nextstep/favorite/application/FavoriteService.java

```diff
@@ -37,7 +38,7 @@ public FavoriteService(FavoriteRepository favoriteRepository, MemberService memb
     @Transactional
     public Long createFavorite(final LoginMember loginMember, final FavoriteRequest favoriteRequest) {
         Member member = memberService.findMemberByEmail(loginMember.getEmail());
-        pathFinder.retrieveStationPath(DISTANCE.getValue(), favoriteRequest.getSource(), favoriteRequest.getTarget());
+        pathFinder.retrieveStationPath(Optional.ofNullable(null), DISTANCE.getValue(), favoriteRequest.getSource(), favoriteRequest.getTarget());
```

Comment:
```suggestion
        pathFinder.retrieveStationPath(null, DISTANCE.getValue(), favoriteRequest.getSource(), favoriteRequest.getTarget());
```
무조건 null을 넘겨주는 상황에서 Optional.ofNullable()의 사용은 무의미해 보이네요.